client/blocks/login/login-form.jsx

```diff
@@ -119,13 +128,10 @@ export class LoginForm extends Component {
 		this.usernameOrEmail = input;
 	};
 
-	linkSocialUser = ( service, usernameOrEmail ) => {
-		this.setState( {
-			usernameOrEmail: usernameOrEmail,
-			linkingSocialUser: true,
-			linkingSocialService: capitalize( service ),
-		} );
-	};
+	isLinkingSocialUser() {
```

Comment:
What does "linking social user" here do? Are we linking an external service account to a WP.com account?

(It was a little confusing to see that this information was derived from create social account errors.)